roles-in-tech/security.md

```diff
@@ -87,29 +87,29 @@ These are some misconceptions that people have in their mind but it is not alway
 
 - **Strong passwords are good enough** - It is not necessary to have a strong password but it is also important that how strongly you have designed. i.e. use of special characters or alphanumeric characters. Adding more than one layer of security better than only setting passwords.
 
-- **Antivirus will secure my computer** - It is one of the most common misconception. But as virus is a piece of code, anti-virus is also the piece of code. They both are created by those having knowledge of both and to fight against each other and competitive to each other.
+- **Antivirus will secure my computer** - It is one of the most common misconceptions. But a virus is a piece of code, anti-virus is also the piece of code. They both are created by those having knowledge of both and to fight against each other and competitive with each other.
 
 - **Cyber Security is just a form of defense** - “The best defense is a good offense.”
 
 - **You will never be attacked** - Unfortunately, the attacked business is often the last to know that it's been breached. It's only after banks discover fraudulent charges that businesses hear about a suspected hack.
 
-- **Hackers are criminals** - It is a big misconception. There are type of hackers and not all hackers are criminals. There are three types of hackers called - *White Hat Hackers(Who 'hacks' legally)* , *Grey Hat Hackers(who may sometimes violate laws or typical ethical standards, but do not have the malicious intent)* and *Black Hat Hackers(Illegal hacking)*.
+- **Hackers are criminals** - It is a big misconception. There are the type of hackers and not all hackers are criminals. There are three types of hackers called - *White Hat Hackers(Who 'hacks' legally) *, *Grey Hat Hackers(who may sometimes violate laws or typical ethical standards, but do not have the malicious intent)* and *Black Hat Hackers(Illegal hacking)*.
```

Comment:
`the` is not required and the space before the asterisk(*) is not required.

Reply:
done